### DISCUSSION TITLE
Invalid configuration encountered when adding the domain name

### DISCUSSION THREAD

**Discussion by AnnaZou1103** [vercel/community#7177] [Help]
### Summary

I am currently trying to add my own domain name to a deployment on the Vercel platform. The website requires me to add a CNAME record on my DNS provider. Following the provided instructions, I have added the following record:

> Type: CNAME
> Name: eval.cs
> Value: cname.vercel-dns.com

Despite this, the website still indicates that the configuration is invalid. Could anyone provide some suggestions on how to resolve this issue?

### Steps to Reproduce

 I have attached screenshots of the provided instructions and the records I tried to add for reference.
![Screenshot 2024-06-12 at 15 53 54](https://github.com/vercel/community/assets/89054199/5b8b67c1-ac12-49a4-9bca-88c98c418147)
![IMG_3529](https://github.com/vercel/community/assets/89054199/b9865018-2df8-46c9-bf19-808b49052d00)

**Comment by amyegan** [comment]
Hey @AnnaZou1103. We have some domain troubleshooting tips posted at https://vercel.com/docs/projects/domains/troubleshooting

[a comment by an automated account is not shown]
